### ISSUE TITLE
Resource Points

### ISSUE THREAD

**Issue by aeauseth** [dmdorman/hero6e-foundryvtt#1630]
Kraven Kor — Today at 11:49 AM
I believe the source book is Dark Champions 6E.  I know I have seen it mentioned in other books, but none that I own have the "actual rules" for such in them.
HERO Designer has them as an option under Perks.
Resource Pools can be "Equipment," "Vehicles & Bases," "Followers and Contacts," or "Miscellaneous" per the HD dropdowns.
Typical 1 AP for 5 Real Points setup for all.
Think of them as VPP that only cares about the Real Points, and has "Can only change 'between missions' or at base, etc."
And limited to fairly generic stuff vs. self-built powers.
In other words, "Gun from the Star HERO Book" is fine, but "AVAD DOES BODY Drain Gun that I built with points" would not be.
